client/src/components/SelectsGroup/SelectsGroup.js

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import DropDownSelect from '../DropDownSelect/DropdownSelect';
+import PropTypes from 'prop-types';
+import './SelectsGroup.scss';
+
+const propTypes = {
+  options: PropTypes.object,
+  date: PropTypes.string,
+  stockCode: PropTypes.string,
+  handleChange: PropTypes.func,
+};
+const defaultProps = {
+  options: {
+    stockCodeOptions: [],
+    dateOptions: [],
+  },
+  date: '',
+  stockCode: '',
+  handleChange: () => {},
+};
+
+const SelectsGroup = (props) => {
+  return (
+    <div className='selects-group-container'>
+      <DropDownSelect
+        value={props.stockCode}
+        title='代號'
+        changed={props.handleChange}
+        options={props.options.stockCodeOptions}
+        class='stock-code-select'
+        type='stock_code'
```

Comment:
When `prop` type is string, please wrap as double quote `""`.

https://github.com/airbnb/javascript/tree/master/react#quotes